distributed/scheduler.py

```diff
@@ -5727,11 +5728,7 @@ def add_plugin(
 
         self.plugins[name] = plugin
 
-    def remove_plugin(
-        self,
-        name: str | None = None,
-        plugin: SchedulerPlugin | None = None,
-    ) -> None:
+    def remove_plugin(self, name: str | None = None) -> None:
```

Comment:
I took out the plugin parameter since it's not used and not documented.

Reply:
`plugin` has been deprecated for a long time and it looks like any deprecated functionality was removed in https://github.com/dask/distributed/pull/6144. To me, it looks like it was left by accident, so removing it should be fine.